### DISCUSSION TITLE
Need help to understand how this work

### DISCUSSION THREAD

**Discussion by paolofrancioso** [lucidrains/big-sleep#134] [Q&A]
solved

**Comment by newbehnam** [comment]
끄다 저항
 커패시터끄다
 다이오드끄다
트랜지스터끄다
 전임자끄다
 집적 회로 또는 IC끄다
 마이크로컨트롤러끄다
 변압기끄다
 퓨즈끄다
릴레이끄다
 키끄다
 엔진끄다
 회로 차단기끄다
 LCD 유형끄다
 나끄다

**Comment by newbehnam** [comment]
끄다 저항
 커패시터끄다
 다이오드끄다
트랜지스터끄다
 전임자끄다
 집적 회로 또는 IC끄다
 마이크로컨트롤러끄다
 변압기끄다
 퓨즈끄다
릴레이끄다
 키끄다
 엔진끄다
 회로 차단기끄다
 LCD 유형끄다
 나끄다
